pom.xml

```diff
@@ -115,6 +115,10 @@
     <url>https://builds.apache.org/view/A-D/view/Accumulo/</url>
   </ciManagement>
   <properties>
+    <accumulo.anyClusterTests>org.apache.accumulo.test.categories.AnyClusterTest</accumulo.anyClusterTests>
+    <accumulo.integrationTestsExcludedGroups />
+    <accumulo.integrationTestsGroups>${accumulo.anyClusterTests},${accumulo.miniclusterTests}</accumulo.integrationTestsGroups>
```

Comment:
I just [proposed a feature enhancement][1] to the maven-surefire-plugin and maven-failsafe-plugin to separate these properties as `tests.*` and `it.*`. If that proposal is accepted, we should use whatever property names the maven plugin developers agree on, just to simplify things and reduce the changeset in the future when the plugin is updated to a version which supports those.

[1]: https://lists.apache.org/thread.html/fc81af97b06d7f6e2c69a795750adb5a5de6dec348ed608041751b65@%3Cdev.maven.apache.org%3E